internal/controller/drplacementcontrol_controller.go

```diff
@@ -50,6 +50,10 @@ const (
 	// PlacementDecisionName format, prefix is the Placement name, and suffix is a PlacementDecision index
 	PlacementDecisionName = "%s-decision-%d"
 
+	// PlacementDecisionReasonFailoverRetained is a reason added to retain a cluster decision that the DRPC
+	// failed over from. It is to aid deletion of the decision once VRG is noted as Secondary on the cluster
+	PlacementDecisionReasonFailoverRetained = "RetainedOnWorkloadFailover"
```

Comment:
"RetainedForFailover" will be more clear.

Can we add this to the api so we don't need to copy it in e2e?

Reply:
Changed the string.

I am unsure yet if we want to add it to the API. We can and should, I would possibly do this after some time, before declaring it as an API artifact. We really do not use this in our CRs it is the OCM CR that we place this value on, hence hesitant to declare an API constant for an non-DR CR at present.